x-pack/plugins/security_solution/public/value_list/translations.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ADD_LIST_ITEM = i18n.translate('xpack.securitySolution.listItems.addListItem', {
+  defaultMessage: 'Add list item',
+});
+
+export const SUCCESSFULLY_ADDED_ITEM = i18n.translate(
+  'xpack.securitySolution.listItems.successfullyAddedItem',
+  {
+    defaultMessage: 'Successfully added list item',
+  }
+);
+
+export const VALUE_REQUIRED = i18n.translate('xpack.securitySolution.listItems.valueRequired', {
+  defaultMessage: 'Value is required',
+});
+
+export const VALUE_LABEL = i18n.translate('xpack.securitySolution.listItems.valueLabel', {
+  defaultMessage: 'Value',
+});
+
+export const ADD_VALUE_LIST_PLACEHOLDER = i18n.translate(
+  'xpack.securitySolution.listItems.addValueListPlaceholder',
+  {
+    defaultMessage: 'Add list item..',
```

Comment:
We should probably get a copy pass on these; EUI has some helpful guidelines; I don't know where ellipses (or two periods 😅 ) fall within them.

Reply:
++

Reply:
I'd need to see this copy in the UI to make a suggestion. Our [current guidelines](https://eui.elastic.co/#/guidelines/writing/guidelines#punctuation) state that ellipses are acceptable to use after truncated text or situations that require waiting, but advise against adding them to the end of actions.